scripts/prepdocslib/integratedvectorizerstrategy.py

```diff
@@ -0,0 +1,189 @@
+from typing import Optional
+
+# Workaround to use the preview SDK
```

Comment:
So do we need to stay on the preview release to use this? I was thinking of updating to the latest stable SDK, which changes some of the parameter names, but I won't do that if its incompatible with this.